Challenges/bank_account.py

```diff
@@ -0,0 +1,22 @@
+class BankAccount:
```

Comment:
The package name `Challenges` should also be in lowercase as mentioned [here](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)